assets/js/header.js

```diff
@@ -1,25 +1,29 @@
-const languages = document.querySelectorAll(".lang");
-languages.forEach((item) => {
-  item.addEventListener("click", function () {
-    languages.forEach((item) =>
-      item.classList.remove(
+function switchCSSLanguageButtons() {
+  const languages = document.querySelectorAll(".lang");
```

Comment:
```suggestion
  const languages = document.querySelectorAll(".lang");
  const activeLanguageClassList = [
    "px-3",
    "py-0.5",
    "bg-black",
    "text-white",
    "rounded",
    "border-solid",
    "border"
  ]
```
E đặt 1 biến array để dùng lại classList nha